lib/Makefile.am

```diff
@@ -17,11 +17,11 @@
 #
 MAINTAINERCLEANFILES    = Makefile.in
 
-EXTRA_DIST		= pacemaker.pc.in pacemaker-pe_rules.pc.in pacemaker-pe_status.pc.in $(target_LIBS:%=%.in)
+EXTRA_DIST		= pacemaker.pc.in pacemaker-pe_rules.pc.in pacemaker-pe_status.pc.in pacemaker-internal.pc.in $(target_LIBS:%=%.in)
```

Comment:
Blech, I forgot about the existing pacemaker.pc.in (for libcrmcommon). Since the new library will have both public and private APIs, I don't want to mark it internal -- maybe libpacemaker.pc.in?